subsys/mpsl/fem/Kconfig

```diff
@@ -47,10 +56,21 @@ config MPSL_FEM_NRF21540_GPIO
 	select NRFX_GPIOTE
 	select NRFX_PPI if SOC_SERIES_NRF52X
 	select NRFX_DPPI if SOC_SERIES_NRF53X
+	select MPSL_FEM_NCS_SUPPORTED_FEM_USED
 	bool "nRF21540 front-end module in GPIO mode"
 	help
 	  FEM device is nRF21540 and its control mode is GPIO.
 
+config MPSL_FEM_NRF21540_GPIO_SPI
+	depends on MPSL_FEM_NRF21540_GPIO_SPI_SUPPORT
+	select EXPERIMENTAL
+	select NRFX_GPIOTE
+	select NRFX_PPI if SOC_SERIES_NRF52X
+	select NRFX_DPPI if SOC_SERIES_NRF53X
```

Comment:
missing `select MPSL_FEM_NCS_SUPPORTED_FEM_USED`

Reply:
I forgot to save and add the file to the commit, thanks for noting that

Reply:
Fixed